frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx

```diff
@@ -32,6 +32,7 @@ export const ProjectHealth = () => {
     const {
         data: { averageHealth },
     } = useProjectStatus(projectId);
+    const averageHealthRounded = Math.round(averageHealth);
```

Comment:
Sorry as this was my PR, but I think we should do it in backend.

Reply:
Yeah, I don't mind that at all. I'll update the PR.